asklib.module

```diff
@@ -124,6 +124,43 @@ function asklib_cron() {
       $question->save();
     }
   }
+
+  // Clear contact information from 2 year old questions every day after 1 am.
+  $request_time = \Drupal::time()->getRequestTime();
+  $next_execution = \Drupal::state()->get('asklib.next_question_contact_cleanup', 0);
+
+  if($request_time > $next_execution) {
+    _asklib_clear_old_question_contact_info();
+
+    \Drupal::state()->set('asklib.next_question_contact_cleanup',
+    mktime(1, 0, 0, date("m")  , date("d")+1, date("Y")));
+  }
+  
+}
+
```

Comment:
This could use a function comment

Reply:
I added a small description to the function comment.